config/other.go

```diff
@@ -205,8 +205,8 @@ func setDefaults(conf *Config, bus awlevent.Bus) {
 	}
 	conf.emitter = emitter
 
-	if conf.Update.UpdateServerURL == "" {
-		conf.Update.UpdateServerURL = "http://example/example.json" // TODO replace for real server
+	if u := conf.Update.UpdateServerURL; u == "" || u == "http://example/example.json" {
```

Comment:
as long as only we are awl users does debug url check has any sense?

Reply:
I have about 6 devices with this config set and I'm a bit lazy to change it manually. We can remove this check in next releases

Reply:
kek)